dts/arm/nxp/nxp_rt118x.dtsi

```diff
@@ -90,6 +90,22 @@
 		gpio-controller;
 		#gpio-cells = <2>;
 	};
+
+	/* GPT1 is used for the hardware timer, not as a standard counter */
```

Comment:
Is this the case here? IE has GPT1 been tested as a system clock source (which is what we use hardware timers for)

Reply:
No, GPT1 as HW timer isn't be tested, I only refer other RT platforms. 
I have modified this gpt to GPT1, if If there is a need for HW TIMER in the future, I will change GPT1 to HW TIMER.